core/tag_api.php

```diff
@@ -920,19 +920,12 @@ function tag_display_link( array $p_tag_row, $p_bug_id = 0 ) {
  */
 function tag_display_attached( $p_bug_id ) {
 	$t_tag_rows = tag_bug_get_attached( $p_bug_id );
-
-	if( count( $t_tag_rows ) == 0 ) {
-		echo lang_get( 'tag_none_attached' );
-	} else {
-		$i = 0;
-		foreach( $t_tag_rows as $t_tag ) {
-			echo( $i > 0 ? config_get( 'tag_separator' ) . ' ' : '' );
-			tag_display_link( $t_tag, $p_bug_id );
-			$i++;
-		}
+	$i = 0;
+	foreach( $t_tag_rows as $t_tag ) {
+		echo( $i > 0 ? config_get( 'tag_separator' ) . ' ' : '' );
```

Comment:
I know this is old code, but let's get the config for the separate before the loop into a temp variable.